Index: plasma-widget-volumewin7mixer-19/org.kde.plasma.volumewin7mixer/package/contents/ui/main.qml
===================================================================
--- plasma-widget-volumewin7mixer-19.orig/org.kde.plasma.volumewin7mixer/package/contents/ui/main.qml
+++ plasma-widget-volumewin7mixer-19/org.kde.plasma.volumewin7mixer/package/contents/ui/main.qml
@@ -40,9 +40,9 @@ Item {
 
     AppletConfig { id: config }
 
-    // Layout.minimumHeight: units.gridUnit * 12
+    Layout.minimumHeight: Layout.preferredHeight
     Layout.preferredHeight: config.mixerGroupHeight + (mediaControllerVisible ? config.mediaControllerHeight : 0)
-    Layout.minimumWidth: 10
+    Layout.minimumWidth: Layout.preferredWidth
     Layout.preferredWidth: mixerItemRow.width
     Layout.maximumWidth: plasmoid.screenGeometry.width
     property int maxVolumePercent: 100.0 // plasmoid.configuration.maximumVolume
@@ -546,4 +546,4 @@ Item {
 
         // plasmoid.action("configure").trigger();
     }
-}
\ No newline at end of file
+}
